assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/config/DataCiteConfigurationFactoryTest.java

```diff
@@ -44,40 +54,76 @@ void setUp() {
         setupSystemUnderTest();
     }
 
+    @Test
+    void constructorWithExampleConfigAsInputstreamThenContains2KnownCustomers()
```

Comment:
describe the input, i.e., contstructorCreates2CustomersWhenConfigHasTwoValidAndOneInvalidEntry

Reply:
Fixed